src/presenters/admin/sidebar-presenter.php

```diff
@@ -81,24 +81,24 @@ class="attachment-full size-full content-visible"
 				<div class="yoast-sidebar__section">
 					<h2>
 						<?php
-						esc_html_e( 'Learn SEO', 'wordpress-seo' );
+						\esc_html_e( 'Learn SEO', 'wordpress-seo' );
 						?>
 					</h2>
 					<p>
 						<?php
 						$academy_shortlink = WPSEO_Shortlinker::get( 'https://yoa.st/3t6' );
 
 						/* translators: %1$s expands to Yoast SEO academy, which is a clickable link. */
-						printf( esc_html__( 'Want to learn SEO from Team Yoast? Check out our %1$s!', 'wordpress-seo' ), '<a href="' . esc_url( $url ) . '"><strong>Yoast SEO academy</strong></a>' );
+						\printf( \esc_html__( 'Want to learn SEO from Team Yoast? Check out our %1$s!', 'wordpress-seo' ), '<a href="' . \esc_url( $buy_yoast_seo_shortlink ) . '"><strong>Yoast SEO academy</strong></a>' );
```

Comment:
Shouldn't this be `$academy_shortlink` instead of `$bug_yoast_seo_shortlink`? The link label seems to suggest so 😅